app/tests/unit/routes/v1/rbac.spec.js

```diff
@@ -327,9 +329,13 @@ describe(`GET ${basePath}/submissions`, () => {
 describe(`PUT ${basePath}/submissions`, () => {
 
   it('should return 200', async () => {
+    formService.read = jest.fn().mockReturnValue([]);
     // mock a success return value...
     service.modifySubmissionUser = jest.fn().mockReturnValue([]);
 
+    emailService.submissionAssigned = jest.fn().mockReturnValue([]);
+    emailService.submissionUnassigned = jest.fn().mockReturnValue([]);
```

Comment:
Looks like these were newly added in. Make sure when writing unit tests for things that need to be mocked that they are always `mockReset` or `mockRestore` after each unit test executed; otherwise you will have very strange side effects that are hard to catch in the future as more tests are added to this suite.
Ref: https://jestjs.io/docs/mock-function-api#mockfnmockreset

Reply:
right, I have implemented a mockReset after the asserts of this test since this is the only location we are mocking the submissionAssigned and submissionUnassigned functions.